api/proto/info/info.proto

```diff
@@ -76,16 +76,48 @@ enum IPhyIoType {
         IPhyIoOther = 255;
 }
 
-// Information about assignable I/O adapter bundles
+// ZioBundle - Information about assignable I/O adapter bundles
+// XXX - Does it represent a Bundle or each Member?
```

Comment:
This is old and represents a bundle. If we need to echo back everything which was in the member configation in phyio then we need to define a new ZioMember. For instance, we want a place to be able to report that PCI ID X:Y does not exist. I don't think we can do that in this ZioBundle message.

But changing ZioBundle doesn't make sense to me.

Reply:
Ok.. I will add ZioMember .. Will leave the Bundle alone.